bc_obps/service/tests/test_user_service.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from registration.models.app_role import AppRole
+from registration.tests.utils.bakers import (
+    user_baker,
+    user_operator_baker,
+)
+from registration.constants import UNAUTHORIZED_MESSAGE
+from registration.models.user import User
+from registration.models.user_operator import UserOperator
+from registration.tests.utils.bakers import operator_baker
+from model_bakery import baker
+from service.user_service import UserService
+
+pytestmark = pytest.mark.django_db
+
+
+class TestUserService:
+    @staticmethod
+    def test_get_user_if_authorized_cas_user_success():
+        admin_user = baker.make(User, app_role=AppRole.objects.get(role_name="cas_analyst"))
+        target_user = user_baker()
+        result = UserService.get_if_authorized(admin_user.pk, target_user.pk)
+        assert result == target_user
+
+    @staticmethod
+    def test_get_user_if_authorized_industry_user_success():
```

Comment:
could we please add another test case here to assert success that when there's multiple users associated with the operator, that all relevant user_operators are fetched? I want to make sure that at some point in the future we don't accidentally use `.first()` instead of `.all()` in the query

Reply:
Sorry I don't get it 😄  I'm pretty sure my brain is not functioning properly right now, but could you please explain more? Thanks!

Reply:
ahh sorry! I misread the code. I thought this was getting all users associated with the admin_user's operator. I can see now it's fetching a specific user associated with admin_user's operator. My bad!